ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -842,4 +834,53 @@ sub relativize_url {
     return $rel_path;
 }
 
+sub get_header {
+	my $mode = shift() // "html";
+	my ($title, $csslink, $bodyid, $block) = @_;
+
+	my $ret = "";
+
+	if ($mode eq "xhtml") {
+		$ret = qq(<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+		<title>$title</title>$csslink
+		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
+	</head>
+
+<body$bodyid>
+$block);
+	} elsif ($mode eq 'html') {
+		$ret = qq(<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+		<title>$title</title>$csslink
+	</head>
+
+<body$bodyid>
+$block);
+	}
+
+	return $ret;
+}
+
+sub get_footer {
+	my $mode = shift() // "html";
+	my ($title, $csslink, $bodyid, $block) = @_;
+
+	my $ret = "";
+
+	# Modes are the same (for now) for footer stuff
+	if ($mode eq "xhtml") {
+		$ret = "$block\n</body>\n\n</html>";
+	} elsif ($mode eq 'html') {
+		$ret = "$block\n</body>\n\n</html>";
+	}
```

Comment:
I read this twice (first when reviewing the entire changeset and again when reviewing this single commit) and twice I stared at this trying to understand what the difference is between the content strings until I realized they are exactly the same. (Both times I skipped the comment line.)

And it's true that it's well past midnight here, but I doubt having the same line of code twice with these conditions is a good idea.

```perl
	if ( $mode eq 'xhtml' || $mode eq 'html' ) {
		$ret = "$block\n</body>\n\n</html>";
	}
```

This is easier to read and allows the author to easily refactor this in the future, which I think is the intention.

Reply:
This was just forward thinking on my part... Eventually the HTML5 and XHTML versions will probably be different so I broke them out to prepare for that. You are 100% right, that currently they're identical. If you feel very strongly about it I can merge them.